### ISSUE TITLE
Incompatible decompression

### ISSUE THREAD

**Issue by gnoymmij** [kesla/node-snappy#57]
Hi,

I have a python script that uses snappy to compress JSON and push it to a redis queue. When I obtain the compressed JSON from the queue in nodejs, the decompress function throws an error stating that a Buffer is needed. 

Upon further investigation, it seems that the python snappy library's compressed data is of type str. In nodejs, the obtained compressed data from redis is also of type String.

Is there a work around for this?
Please advise.

Thanks.

**Comment by kesla** [comment]
@gnoymmij hey!

Snappy assumes that it'll get a raw buffer when doing decompression. When you do `redis.get` in node.js it gets converts to strings as default but what you'd like to get is a Buffer, see `return_buffers`-options in https://github.com/NodeRedis/node_redis#rediscreateclient.

**Comment by gnoymmij** [comment]
@kesla Thanks for the tip!

I have modified the redis client and checking the type obtained from the queue now returns a Buffer. snappy.isValidCompressedSync(input) also returns true.

However, I then encountered the follow error: SyntaxError: Unexpected token �
From python, I just make a simple compress and did nothing else. Could it be a snappy C library issue? As I installed snappy using the OS repo (libsnappy-dev). The python library I'm using is this: https://github.com/andrix/python-snappy

Appreciate your input.
